src/utils/defaultLanguageStrings.ts

```diff
@@ -133,7 +133,8 @@ export default {
   FILTER_START_TYPING: 'Start typing...',
   FILTER_SEARCH_LIST: 'Search list...',
   FILTER_SORT_BY: 'Sort by',
-  FILTER_RESET_SORT: 'Reset sort',
+  FILTER_RESET_SORT: 'Reset column sort',
+  FILTER_RESET_PAGINATION: 'Reset pagination',
```

Comment:
@nmadruga You might want to pre-translate these into the Angola localisations file so they're already there when this PR is merged.